frontend/src/components/Forms/Register/Register.tsx

```diff
@@ -1,12 +1,12 @@
 import { useState } from "react";
 import classes from "../Forms.module.scss";
-import Sheet from "../Sheet/Sheet";
+import Sheet from "../../Sheet/Sheet";
 import classNames from "classnames";
 import WarningText from "../WarningText/WarningText";
 import { useNavigate } from "react-router-dom";
 import { EyeOff, Eye } from "lucide-react";
 
-function Register() {
+function LoginPage() {
```

Comment:
seems like there's some conflicts going on here or something? should be register? idk maybe im missing smth but it still says `LogInPage` on my end